packages/nimble-components/src/select/testing/select.pageobject.ts

```diff
@@ -268,4 +268,25 @@ export class SelectPageObject {
         }
         return this.selectElement.filterInput;
     }
+
+    private getClearButton(): Button | null | undefined {
+        if (!this.selectElement.clearable) {
+            throw new Error(
+                'Select must set "clearable" in order to click clear button'
+            );
+        }
+
+        if (
+            this.selectElement.selectedIndex === -1
+            || this.selectElement.displayPlaceholder
```

Comment:
Perhaps, but at the moment the only place making this check is the page object, so making a new property, even an internal one, seems unnecessary at the moment. I'd rather not create APIs just for the sake of testing clarity, particularly when the intent is fairly clear.

Reply:
Personally, from a readability/"clean code" standpoint, I do't think adding extra 1-or-2-line internal functions is something to avoid, but given it's only used in the page object in this case, it's not a big deal either way. 👍